app/controllers/calendar_controller.rb

```diff
@@ -74,16 +74,14 @@ def simple_404
 
   def handle_bank_holiday_ics_calendars
     if scope == "bank-holidays"
-      i18n_backend.send(:init_translations) unless i18n_backend.initialized?
-      translations = i18n_backend.send(:translations)
-      division_key = translations[I18n.locale][:common][:nations].key(params[:division]).to_s
-      params[:division] = "common.nations." + division_key
-    end
-  end
+      division_slug = {
```

Comment:
Indeed, going to add the test for those 6.
Which class do you suggest adding the the constant?